envisage/tests/test_service_registry.py

```diff
@@ -34,7 +34,7 @@ def setUp(self):
         # the same interface!
         self.service_registry = Application(service_registry=ServiceRegistry())
 
-        # module 'foo' need to be cleared out when this test is run by nose,
+        # module 'foo' need to be cleared out when this test is run,
         # because other tests also import foo.
```

Comment:
i think this comment is still relevant, even if it isnt nose that is running the tests.